static/static_file_modifier.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2015 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package static provides a modifier that allow Martian to reurn static files
+// local to Martian.
+package static
+
+import (
+	"encoding/json"
+	"mime"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/google/martian"
+	"github.com/google/martian/parse"
+)
+
+type staticModifier struct {
```

Comment:
Most other modifiers are named simply 'Modifier' or 'modifier'.  Do we want to stay consistent with that?